File: LeetCode/2011. Final Value of Variable After Performing Operations/C#/01. Final Value of Variable After Performing Operations.cs
int FinalValueAfterOperations(string[] operations)
{
    int initValue = 0;
    foreach (string operation in operations)
    {
        if (operation == "--X" || operation == "X--")
        {
            initValue--;
        }
        else
        {
            initValue++;
        }
    }
    return initValue;
}

File: LeetCode/2011. Final Value of Variable After Performing Operations/JavaScript/01. Final Value of Variable After Performing Operations.js
var finalValueAfterOperations = function (operations) {
    let initValue = 0;
    operations.forEach(operation => {
        if (operation == "--X" || operation == "X--") {
            initValue--;
        }
        else {
            initValue++;
        }
    });
    return initValue;
};